provision/acc_provision/test_main.py

```diff
@@ -297,6 +297,19 @@ def test_list_flavors_msg():
             assert tmperr.read() == stderr.read()
 
 
+@in_testdir
+def test_overlapping_subnets():
+    try:
+        run_provision(
+            "with_overlapping_subnets.inp.yaml",
+            None,
+            None
+        )
+    except SystemExit:
+        # expected to exit with errors
+        pass
```

Comment:
Thanks for adding the UT, but how does confirm that the exception was thrown? Would the test not pass even if exception was not thrown? I believe need to validate that the exception was thrown by asserting the exception string on stderr. Please see the prior test cases.

Reply:
Addressed.